NuGet.config

```diff
@@ -3,13 +3,9 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="myget-fxlab" value="https://dotnet.myget.org/F/dotnet-corefxlab/api/v3/index.json" />
```

Comment:
What is the reason for deleting these other feeds?  We typically like them around as a reference for developers (and for us as well).

Reply:
1. We're trying to get off of myget
2. The number of feeds has an effect on nuget performance, so if this doesn't have packages that are actively in use by the repo we should remove the feeds. The goal is to get down to just azdo feeds and nuget,.org.